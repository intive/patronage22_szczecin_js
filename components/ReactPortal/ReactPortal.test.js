/* globals describe, it, expect  */

import { render } from '@testing-library/react'
import ReactPortal from './ReactPortal'

describe('ReactPortal', () => {
  it('should render ReactPortal correctly', () => {
    const { baseElement } = render(<ReactPortal wrapperId='testId' />)

    expect(baseElement).toMatchSnapshot()
  })
})
