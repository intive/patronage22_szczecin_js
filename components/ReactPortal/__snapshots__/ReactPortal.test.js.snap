// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReactPortal should render ReactPortal correctly 1`] = `
<body>
  <div />
  <div
    id="testId"
  />
</body>
`;
